### ISSUE TITLE
Debugger prints error message when using `p` in self-destruct tx

### ISSUE THREAD

**Issue by haltman-at** [trufflesuite/truffle#4566]
## Issue

Suppose you're trying to debug a transaction where a contract successfully self-destructs... well, at the moment, for the most part, you can't, but that's a separate issue (and we're not going to fix that until Ganache 7.1 is out and we've done something about #2894).

But, it's not *completely* true that you can't.  You can still step through with `;` and see *some* of the assembly state, even if it's not all there.  However, now when stepping through with `;` or `p`, you'll get an error message.

We can't at the moment actually print out all the helpful information you would get normally -- that'll have to wait -- but we could at least do something better than erroring.

## Steps to Reproduce

Create a contract that can self-destruct.  Perform a transaction that makes it self-destruct.  Debug that transaction.  Type `p`.

## Expected Behavior

Not an error message.

## Actual Results

```
debug(development:0x912b205d...)> p

  No data on stack.

Instructions:
Uncaught TypeError: Cannot read property 'toString' of undefined
    at ReadStream.emit (events.js:314:20)
    at ReadStream.onkeypress (readline.js:205:10)
    at REPLServer.self._ttyWrite (repl.js:867:9)
    at REPLServer.Interface._ttyWrite (readline.js:1003:14)
    at REPLServer.Interface._line (readline.js:658:8)
    at REPLServer.Interface._onLine (readline.js:329:10)
    at REPLServer.EventEmitter.emit (domain.js:483:12)
    at REPLServer.emit (events.js:314:20)
    at REPLServer.onLine (repl.js:777:10)
    at REPLServer.runBound [as eval] (domain.js:440:12)
    at bound (domain.js:427:14)
    at REPLServer.bound interpreterCallbackified (util.js:210:5)
    at DebugInterpreter.interpreter (/home/sniffnoy/truffle/truffle/packages/core/lib/debug/interpreter.js:617:26)
    at DebugPrinter.printInstruction (/home/sniffnoy/truffle/truffle/packages/core/lib/debug/printer.js:247:39)
    at Object.formatCurrentInstruction (/home/sniffnoy/truffle/truffle/packages/debug-utils/index.js:535:21)
    at Object.formatPC (/home/sniffnoy/truffle/truffle/packages/debug-utils/index.js:549:18)
```

## Environment

* Truffle version (`truffle version`): 5.4.25

**Comment by haltman-at** [comment]
Oops, I forgot to specify: To see the error, you have to press `p` while inside the code that self-destructed, not some other part of the transaction.

**Comment by gnidan** [comment]
This has been addressed in [Truffle v5.4.28](https://github.com/trufflesuite/truffle/releases/tag/v5.4.28). Closing!
